eclipse-che/che-theia
# chore(dogfooding devfile): fix typo in rsync command

Signed-off-by: Vitaliy Gulyy <vgulyy@redhat.com>



### What does this PR do?
Fixes typo in a command, that syncing `che-theia` sources to `/tmp` directory.

### What issues does this PR fix or reference?

No issue, just a simple fixup.

### How to test this PR?

- create a workspace from `che-theia-dev.devsandbox.devfile.yaml` file
- launch 1, 2, 3, 4, 5.. command from `My Workpsace` view
- ensure `che-theia` was built successfully

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### Happy Path Channel

HAPPY_PATH_CHANNEL=stable
